### DISCUSSION TITLE
Is there any possible way to convert my react-js project to solid-js?

### DISCUSSION THREAD

**Discussion by ghost** [solidjs/solid#1202] [Q&A] [answered]
Hi! Is there any possible way to convert my react-js project to solid-js?

**Comment by atk** [comment, marked as answer]
https://github.com/yellowsink/solid-reactor and https://suid.io/tools/react-to-solid are available; both may require a bit of manual work to fix the converted code.

**Reply by ghost** [reply]
@atk Hi! thank you for feedback.
